pages/ambassadors/index.page.tsx

```diff
@@ -68,10 +68,10 @@ export default function communityPages() {
             data-testid='Ambassadors-contributions'
           >
             <h1 className='text-h3 font-bold text-gray-900 dark:text-white'>
-              JSON Ambassador Contributions
+              JSON Schema Ambassador Contributions
             </h1>
             <p className='mt-5 text-lg text-gray-700 dark:text-slate-100'>
-              JSON Ambassadors are passionate about JSON Schema. They share
+              Ambassadors are passionate about JSON Schema. They share
               their interest, expertise, and excitement within their communities
               to help others build better software.
```

Comment:
```suggestion
              Ambassadors are passionate about JSON Schema. They share their
              interest, expertise, and excitement within their communities to
              help others build better software.
```

Reply:
My changes didn't touch this . Why is it failing on my pr?

Reply:
> My changes didn't touch this . Why is it failing on my pr?

You removed JSON from the sentence, which altered the formatting and now the linter is failing.